src/index.js

```diff
@@ -1,9 +1,5 @@
-import { createFactory } from '@volusion/element-block-utils';
-import { StarterBlockFactory } from './Block';
-import { getStyles } from './getStyles';
-import { getConfigSchema, defaultConfig } from './configs';
+import { StarterBlock as block } from './Block';
```

Comment:
**Suggestion:**
@cameron-jones-volusion I've been thinking about this, and how we shouldn't have to update the export name on every block. How about updating the Block.js export to default and the import to support that?